common/autoware_overlay_rviz_plugin/autoware_minimap_overlay_rviz_plugin/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_minimap_overlay_rviz_plugin</name>
+  <version>0.1.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="khalil@leodrive.ai">khalil</maintainer>
+  <license>TODO: License declaration</license>
+
+  <depend>angles</depend>
+  <depend>nav_msgs</depend>
+  <depend>rclcpp</depend>
+  <depend>rcpputils</depend>
+  <depend>rviz_common</depend>
+  <depend>rviz_default_plugins</depend>
+  <depend>sensor_msgs</depend>
+  <depend>tf2_ros</depend>
```

Comment:
Make sure all other removed stuff is here and also add the other packages you depend to here.